pkg/artifacts/paths.go

```diff
@@ -60,20 +74,44 @@ func defaultArtifactsDir() (string, error) {
 	return absPath, nil
 }
 
-// BindFlags binds the artifact related flags.
+// the default is $KUBETEST2_RUN_DIR if set, otherwise ./_rundir
+// constructed as an absolute path to help the ginkgo tester because
+// for some reason it needs an absolute path to the kubeconfig
+func defaultRunDir() (string, error) {
+	if path, set := os.LookupEnv("KUBETEST2_RUN_DIR"); set {
+		absPath, err := filepath.Abs(path)
+		if err != nil {
+			return "", fmt.Errorf("failed to convert filepath from $KUBETEST2_RUN_DIR (%s) to absolute path: %s", path, err)
+		}
+		return absPath, nil
+	}
+
+	absPath, err := filepath.Abs("_rundir")
+	if err != nil {
+		return "", fmt.Errorf("when constructing default rundir, failed to get absolute path: %s", err)
+	}
+	return absPath, nil
+}
+
+// BindFlags binds the artifact and rundir related flags.
 func BindFlags(flags *pflag.FlagSet) error {
 	defaultArtifacts, err := defaultArtifactsDir()
 	if err != nil {
 		return err
 	}
+	defaultRundir, err := defaultRunDir()
+	if err != nil {
+		return err
+	}
 	flags.StringVar(&baseDir, "artifacts", defaultArtifacts, `top-level directory to put artifacts under for each kubetest2 run, defaulting to "${ARTIFACTS:-./_artifacts}". If using the ginkgo tester, this must be an absolute path.`)
+	flags.StringVar(&runDir, "rundir", defaultRundir, `directory to put run related test binaries like e2e.test, ginkgo, kubectl for each kubetest2 run, defaulting to "${KUBETEST2_RUN_DIR:-./_rundir}". If using the ginkgo tester, this must be an absolute path.`)
 	return nil
 }
 
 // MustBindFlags calls BindFlags and panics on an error
 func MustBindFlags(flags *pflag.FlagSet) {
 	err := BindFlags(flags)
 	if err != nil {
-		klog.Fatalf("failed to get default artifacts directory: %s", err)
+		klog.Fatalf("failed to get default artifacts/rundir directory: %s", err)
```

Comment:
```suggestion
		klog.Fatalf("failed to get default artifacts || rundir directory: %s", err)
```
`artifacts/rundir` sounds like it itself could be a dir haha

Reply:
Done :)